package.json

```diff
@@ -48,12 +48,17 @@
     "build:less:cpy": "copyfiles \"src/**/*.less\" \".\" --up 1 --all",
     "build:less:css": "foreach -x \"lessc #{path} #{dir}/#{name}.css\" -i \"**/*.mixin.less\" -g \"modules/{*,*/*,{draft,esl-utils,esl-forms}/*/*}.less\"",
     "build:docs": "copyfiles \"src/**/*.md\" \".\" --up 1 --all",
+    "minify:css": "cleancss -o pages/dist/bundles/localdev.css pages/dist/bundles/localdev.css",
+    "build-pages:dev": "concurrently \"npm run build-pages:ts:dev\" \"npm run build-pages:less:dev\" && npm run build-pages:eleventy:dev",
     "build-pages": "concurrently \"npm run build-pages:ts\" \"npm run build-pages:less\" && npm run build-pages:eleventy",
-    "build-pages:ts": "cd pages && webpack",
-    "build-pages:less": "lessc pages/src/localdev.less pages/dist/bundles/localdev.css --source-map=pages/dist/bundles/localdev.css.map",
+    "build-pages:ts:dev": "cd pages && webpack",
+    "build-pages:ts": "cd pages && webpack --env=production --mode=production",
+    "build-pages:less:dev": "lessc pages/src/localdev.less pages/dist/bundles/localdev.css --source-map=pages/dist/bundles/localdev.css.map",
+    "build-pages:less": "npm run build-pages:less:dev && npm run minify:css",
+    "build-pages:eleventy:dev": "npx @11ty/eleventy",
```

Comment:
Do we not need to pass `env= development` as an attribute here?

Reply:
I've changed 'build-pages' scripts